pallets/providers/src/lib.rs

```diff
@@ -271,6 +275,10 @@ pub mod pallet {
         /// fee for a bucket that doesn't have any files yet)
         #[pallet::constant]
         type ZeroSizeBucketFixedRate: Get<BalanceOf<Self>>;
+
+        /// Trait that has benchmark helpers
+        #[cfg(feature = "runtime-benchmarks")]
+        type BenchmarkHelpers: crate::benchmarking::BenchmarkHelpers<Self>;
```

Comment:
This solution is fine and well executed. Personally, I would have created a trait in `primitives/traits` that does this, then have the Proofs Dealer pallet implement it, and in the conditions of the benchmarks:
```
#[benchmarks(where
	T: crate::Config + pallet_randomness::Config
)]
```

Add a condition that `crate::Config::ProvidersProofSubmitters` should also implement that trait in the runtime `T`. Anyway, I'm fine with this too if you prefer it.